pkg/util/pathutil/util.go

```diff
@@ -11,7 +11,7 @@ import (
 
 const (
 	tmpSuffix      = ".tmp"
-	ownerRW        = 0600
+	ownerRW        = 0606
```

Comment:
The const name seems misleading, given the change we are making here.

Reply:
Changed to `ownerRWOtherRW`